Assessment/CustomerCommand/Console/Command/ImportCustomerCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Assessment\CustomerCommand\Console\Command;
+
+use Assessment\CustomerCommand\Api\ProfileInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use \Magento\Framework\Console;
```

Comment:
```suggestion
use Magento\Framework\Console;
```
No need to first slash here...

Reply:
changed